tests/integration/src/jtag/test_lc_transitions.rs

```diff
@@ -168,35 +170,32 @@ mod test {
             .expect("Failed to connect to the LCC JTAG TAP.");
 
         // Read the LC state.
-        let lc_state = read_lc_state(&mut *tap).expect("Unable to read LC state.");
+        let mut lc_state = read_lc_state(&mut *tap).expect("Unable to read LC state.");
         println!("Initial LC state: {}", lc_state);
         assert_eq!(lc_state, LifecycleControllerState::Prod);
 
         // Perform the RMA LC transition operation.
-        // TODO(caliptra-mcu-sw/issues/454): expect a failure until the PPD pin
-        // is exposed to the FPGA model to enable testing RMA transitions.
-        let result = lc_transition(
+        lc_state = lc_transition(
             &mut *tap,
             LifecycleControllerState::Rma,
             Some(lc_token_to_words(&DEFAULT_LIFECYCLE_RAW_TOKEN.0)),
-        );
-        let err = result.unwrap_err();
-        let lcc_err = err.downcast_ref::<LccUtilError>().unwrap();
-        let status = match lcc_err {
-            LccUtilError::StatusErrors(status) => status,
-            _ => panic!("Expected LccUtilError::StatusErrors, but got {:?}", lcc_err),
-        };
+        )
+        .expect("Unable to transition to RMA.");
+        println!("Post transition LC state: {}", lc_state);
 
-        assert_eq!(
-            *status,
-            LcCtrlStatus::TOKEN_ERROR | LcCtrlStatus::INITIALIZED
-        );
+        // Reset and read the LC state again.
+        model.set_subsystem_reset(true);
+        model.set_subsystem_reset(false);
```

Comment:
nit: You should be able to use HwModel.cold_reset now that it is implemented in `caliptra-sw`

Reply:
ah nice; fixed.